bin/webpack-dev-server.js

```diff
@@ -331,9 +331,12 @@ function processOptions(wpOpt) {
 
 	if(argv["open"] || argv["open-page"]) {
 		options.open = true;
-		options.openPage = argv["open-page"] || "";
```

Comment:
I'm not following you. You're saying that this statement won't be hit because argv is for command line use, yet the statement that my comment was for is within a logical block where the checks are against argv. I'm not sure your reply addresses the comment.